UnitTests/expected/Images-File-Dot-Syntax.swift.out

```diff
@@ -0,0 +1,64 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+#if os(iOS) || os(tvOS) || os(watchOS)
+  import UIKit.UIImage
+  typealias Image = UIImage
+#elseif os(OSX)
+  import AppKit.NSImage
+  typealias Image = NSImage
+#endif
+
+// swiftlint:disable file_length
+// swiftlint:disable line_length
+// swiftlint:disable nesting
+
+struct AssetType: StringLiteralConvertible {
+  private var value: String
+
+  var image: UIImage {
+    return UIImage(asset: self)
+  }
+
+  public init(stringLiteral value: String) {
+    self.value = value
+  }
+
+  public init(extendedGraphemeClusterLiteral value: String) {
+    self.init(stringLiteral: value)
+  }
+
+  public init(unicodeScalarLiteral value: String) {
+    self.init(stringLiteral: value)
+  }
+}
+
+// swiftlint:disable type_body_length
+struct Asset {
+
+  struct Exotic {
+    static let Banana: AssetType = "Exotic/Banana"
+    static let Mango: AssetType = "Exotic/Mango"
+  }
+  static let Lemon: AssetType = "Lemon"
+
+  struct Round {
+    static let Apricot: AssetType = "Round/Apricot"
+    static let Orange: AssetType = "Round/Orange"
+
+    struct Red {
+      static let Apple: AssetType = "Round/Apple"
```

Comment:
Why isn't the value of this generated constant `"Round/Red/Apple"` instead of just `"Round/Apple"`?
Is it because the group in the asset catalogs doesn't have provides-namespace checked, so that's normal and intentional?
In that case I'm wondering if we should create the intermediate structure `struct Red` if it doesn't provide namespace. I'm thinking YES (and so in this case this output is 👍 ), as otherwise the developer wouldn't have created a group in its asset catalog… but just wanted to start the discussion in case anyone had a different opinion.